### PR TITLE
[Keymap] own Lily58 keymap

### PR DESCRIPTION
**Pull request by cykedev**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/lily58/keymaps/cykedev/keymap.c [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+extern uint8_t is_master;
```
Comment (drashna) [inline keyboards/lily58/keymaps/cykedev/keymap.c:33]: This block shouldn't be needed. 
```suggestion
```

File: keyboards/lily58/keymaps/cykedev/keymap.c [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+const char *read_logo(void);
+
+void oled_task_user(void) {
+  if (is_keyboard_master()) {
+    oled_write_ln(read_logo(), false);
+  } else {
+    oled_write_ln(read_logo(), false);
+  }
```
Comment (drashna) [inline keyboards/lily58/keymaps/cykedev/keymap.c:123]: You only need this: 
```suggestion
    oled_write_ln(read_logo(), false);
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by cykedev** [comment]
@drashna I have included your change requests

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**